redisinsight/ui/src/components/home-tabs/HomeTabs.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { EuiTab, EuiTabs } from '@elastic/eui'
+import { useHistory, useLocation } from 'react-router-dom'
+import { Pages, PageValues } from 'uiSrc/constants'
+import { sendEventTelemetry, TelemetryEvent } from 'uiSrc/telemetry'
+import { tabs } from './constants'
+
+import styles from './styles.module.scss'
+
+const HomeTabs = () => {
+  const [activeTab, setActiveTab] = useState('')
+
+  const history = useHistory()
+  const { pathname } = useLocation()
+
+  useEffect(() => {
+    setActiveTab(pathname.startsWith(Pages.rdi) ? Pages.rdi : Pages.home)
+  }, [pathname])
+
+  const onSelectedTabChanged = (path: PageValues, title: string) => {
+    sendEventTelemetry({
+      event: TelemetryEvent.INSTANCES_TAB_CHANGED,
+      eventData: {
+        tab: title
+      }
+    })
+
+    if (path === Pages.rdi) {
+      history.push(Pages.rdi)
+      return
+    }
+
+    history.push(Pages.home)
+  }
+
+  const renderTabs = useCallback(() => tabs.map(({ id, title, path }) => (
+    <EuiTab
+      key={id}
+      isSelected={path === activeTab}
+      onClick={() => onSelectedTabChanged(path, title)}
+      className={styles.tab}
+      data-testid={`home-tab-${id}`}
+    >
+      {title}
+    </EuiTab>
+  )), [activeTab])
+
+  return (
+    <>
```

Comment:
Is a fragment necessary here? Seems like you could just return `<EuiTabs ... >...</EuiTabs>` without `<>...</>`

Reply:
you are right, thanks